sdk/python/generator/Generator.py

```diff
@@ -39,6 +39,10 @@ def generate_files(ast_models, output_directory: Path):
 			'''#!/usr/bin/python
 #
 # Code generated by catbuffer python generator; DO NOT EDIT.
+#
+# pylint: disable=line-too-long, invalid-name, redefined-builtin
```

Comment:
hmm, where does it complain about redefiend builtin?

Reply:
hash